eisbuk-admin/src/components/atoms/AttendanceCard/__tests__/AttendanceCard.test.tsx

```diff
@@ -76,7 +76,7 @@ jest.spyOn(i18n, "t").mockImplementation(((str: string) => str) as any);
 // aliases for thumbs button for easier access
 const thumbsUp = "👍";
 const thumbsDown = "👎";
-const trashCan = "🗑️";
+// const trashCan = "🗑️";
```

Comment:
Unless there's a concrete plan for reintroducing code, let's remove it instead of commenting it out.